src/lib/config/arbitrum/pools.ts

```diff
@@ -45,6 +45,7 @@ const pools: Pools = {
       '0x542f16da0efb162d20bf4358efa095b70a100f9e000000000000000000000436', // tbtc
       '0x567ecfcb22205d279bb8eed3e066989902bf03d5000000000000000000000452', // dola/bb-a-usd
       '0x8bc65eed474d1a00555825c91feab6a8255c2107000000000000000000000453', // dola/usdc
+      '0x481c5fc05d63a58aa2f0f2aa417c021b5d419cb200000000000000000000056a', // reth/bb-a-weth
```

Comment:
This pool doesn't exist.

Reply:
Ah it's on mainnet: https://etherscan.io/address/0x481c5fc05d63a58aa2f0f2aa417c021b5d419cb2#code